automation_tools/__init__.py

```diff
@@ -250,6 +250,11 @@ def setup_default_capsule(interface=None, run_katello_installer=True):
         print('Was not possible to fetch domain information')
         sys.exit(1)
 
+    dhcp_range = os.environ.get('DHCP_RANGE')
```

Comment:
Wait. so we're not required to provide it? What if I don't want to do it? Couldn't we make it optional? << @kbidarkar @sghai